### PR TITLE
Wrap locale files in UMD declarations

### PR DESCRIPTION
**Pull request by m90**
This would solve #318 

The rake task is kind of hacky, but probably still better than doing it manually.

### CODE REVIEW

**Comment by rmm5t** [comment]
🍻 Thanks

File: locales/jquery.timeago.af.js [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+    day: "1 dag",
+    days: "%d dae",
+    month: "1 maand",
+    months: "%d maande",
+    year: "1 jaar",
+    years: "%d jaar",
+    wordSeparator: " ",
+    numbers: []
+  };
+});
```
Comment (cocochepeau) [inline locales/jquery.timeago.af.js]: Should be:
```
}));
```

Or am I wrong? (same goes for the other ones)

Comment (m90) [inline locales/jquery.timeago.af.js]: You are totally right: https://github.com/rmm5t/jquery-timeago/pull/320 - sorry for that.

Comment (rmm5t) [inline locales/jquery.timeago.af.js]: @cocochepeau Thanks for catching this.

**Pull request merged**